src/scalar.rs

```diff
@@ -1186,6 +1183,26 @@ impl UnpackedScalar {
     }
 }
 
+/// Read one or more u64s stored as little endian bytes.
+///
+/// ## Panics
+/// Panics if `src.len() != 8 * dst.len()`.
+fn read_le_u64_into(src: &[u8], dst: &mut [u64]) {
+    assert!(
+        src.len() == 8 * dst.len(),
+        "src.len() = {}, dst.len() = {}",
+        src.len(),
+        dst.len()
+    );
+    for (bytes, val) in src.chunks(core::mem::size_of::<u64>()).zip(dst.iter_mut()) {
```

Comment:
Small thing: you can use `8` here instead of `core::mem::size-of::<u64>()`, since you use `8` in the `assert` above and the `expect` below.

Reply:
Yeah I was thinking it was inconsistent now. Is there an argument for using `size_of` everywhere? It's more "correct" but I don't think there's ever a case(now or in the future) when it will return anything other than 8

Reply:
It's actually fixed at 8 according to the docs https://doc.rust-lang.org/std/mem/fn.size_of.html

But don't worry about it I'm about to push this change, plus 1 extra unit test, and then I'll merge :)

Reply:
Aight, Thanks 🙏